pkg/devicemapper/devmapper.go

```diff
@@ -309,7 +309,7 @@ func FindLoopDeviceFor(file *os.File) *os.File {
 	return nil
 }
 
-// UdevWait wakes any processes that are waiting for udev to complete the specified cookie. (?)
+// UdevWait wakes any processes that are waiting for udev to complete the specified cookie.
```

Comment:
It's more that the kernel may have jobs for the specified cookie and this
call will block until they complete
On Sep 29, 2015 18:24, "Vincent Demeester" <notifications@github.com> wrote:

> In pkg/devicemapper/devmapper.go
> <https://github.com/docker/docker/pull/16642#discussion_r40738302>:
>
> > @@ -309,7 +309,7 @@ func FindLoopDeviceFor(file *os.File) *os.File {
> >  	return nil
> >  }
> >
> > -// UdevWait wakes any processes that are waiting for udev to complete the specified cookie. (?)
> > +// UdevWait wakes any processes that are waiting for udev to complete the specified cookie.
>
> @vbatts <https://github.com/vbatts> that would give "UdevWait waits for
> any processes that are waiting for udev to complete the specified cookie."
> ?
>
> ETOOMUCHWAIT [image: :sweat_smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/16642/files#r40738302>.
>